formfiller/README.md

```diff
@@ -1,24 +1,9 @@
-# formfiler
-[![Build Status](https://travis-ci.org/diy-asylum/formfiller.svg?branch=master)](https://travis-ci.org/diy-asylum/formfiller)
-
-the DIY asylum monorepo
-
-Repo uses the gradle wrapper for its builds
-
-To build and run the tests simply use the included wrapper script
-
-```bash
-./gradlew check
-```
-
-This will build the project, run its tests, and verify the coverage for you.
-
-This repo is configured to enforce Google's code style. I did not pick it for any particular reason.
-I mostly dont like having to think or argue about code style. Spotless will fail anything that does
-not conform to the style, it can also reformat your code automatically.
+# formfiler CLI
 
 This repo contains a CLI application which can provide a filled in I589 form.
 
+It is where any logic involving the PDFs should live.
```

Comment:
I really want the server to barely know about PDFS if at all possible. Just validate the request and send it to methods defined in this module